src/evolutions/src/main/scala/play/api/db/slick/evolutions/EvolutionsModule.scala

```diff
@@ -21,7 +21,7 @@ class EvolutionsModule extends Module {
  */
 trait SlickEvolutionsComponents {
   @deprecated("Use slickApi instead", "3.0.0")
-  def api: SlickApi = slickApi
+  def api: SlickApi
```

Comment:
In this case, since we don't have any implementation of `api` and `slickApi`, the most reasonable thing to do (to avoid breaking changes is):

```scala
@deprecated("Use slickApi instead", "3.0.0")
def api: SlickApi

def slickApi: SlickApi = api
```
So, users that already have an implementation of `api` will get `slickApi` for free, without breaking their trait implementations. In a future release, when we remove `api`, then they need to adapt.

----------

I'm sorry for the back and forth here, but compatibility is an important aspect here. Moreover, thank you for your patience and contribution. :-)